### PR TITLE
Updated Translations for "Never Opened" and "Last Opened"

### PR DESCRIPTION
**Pull request by FareesHussain**
Related issue : #548 
Related PR : #681 
## Explanation
In this pr, I've added the missing string resources and resolved string resource errors

## Checklist

- [x] resolved string resource id ("language_te") error in string.xml-te
- [x] resolved string resource id ("message_duplicate_file_upload_body error
- [x] added string resource for "Never Opened" (in all languages)
- [x] added string resource for "Last Opened" (in all languages)

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Sat Jul 18 05:49:07 IST 2020
+#Thu Nov 12 06:33:49 IST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-bin.zip
```
Comment (cpg) [inline gradle/wrapper/gradle-wrapper.properties]: what's the story with this switch from `all` to `bin`?

Comment (FareesHussain) [reply gradle/wrapper/gradle-wrapper.properties]: I'll revert that

**Comment by Prakhar-Agarwal-byte** [comment]
@FareesHussain Your branch has conflicts with `amahi:beta` branch. You must update your branch with new commits from beta, resolve the conflicts, and then send a PR with the updated branch.

**Comment by FareesHussain** [comment]
Thanks @Prakhar-Agarwal-byte, updated

**Review by Prakhar-Agarwal-byte — COMMENTED** [review]
@FareesHussain can you please add translations of the 2 new strings in German and Hindi as well so that your PR will be complete.

**Comment by FareesHussain** [comment]
@Prakhar-Agarwal-byte Thanks, I've updated it

**Review by Prakhar-Agarwal-byte — APPROVED** [review]

**Review by Prakhar-Agarwal-byte — APPROVED** [review]
@cpg I think this PR is working fine now.

**Review by chirag-jn — APPROVED** [review]

File: src/main/java/org/amahi/anywhere/fragment/AlertDialogFragment.java [hunk 2]
```diff
@@ -29,7 +29,7 @@
     public static final int DUPLICATE_FILE_DIALOG = 1;
     public static final int SIGN_OUT_DIALOG = 3;
     public static final int FILE_INFO_DIALOG = 2;
-    public static final String LAST_OPENED_NULL = "Never opened";
+    public final String LAST_OPENED_NULL = getString(R.string.never_opened);
```
Comment (chirag-jn) [inline src/main/java/org/amahi/anywhere/fragment/AlertDialogFragment.java]: I would recommend that you must remove the reference to `LAST_OPENED_NULL` variable and rather use `getString` directly.

**Review by chirag-jn — CHANGES_REQUESTED** [review]

File: src/main/java/org/amahi/anywhere/fragment/AlertDialogFragment.java (hunk 2 above)
Comment (FareesHussain) [reply src/main/java/org/amahi/anywhere/fragment/AlertDialogFragment.java]: @chirag-jn 
Done, sorry for the delay.

**Review by Prakhar-Agarwal-byte — APPROVED** [review]

**Review by chirag-jn — APPROVED** [review]
I reformatted the changed files so that they match the coding conventions. Rest looks good.

**Comment by FareesHussain** [comment]
> I reformatted the changed files so that they match the coding conventions. Rest looks good.

Thanks a lot

**Comment by FareesHussain** [comment]
@cpg Please have a look

**Comment by cpg** [comment]
Oh no. Prior PR merges caused two conflicts. They look easy to merge, however I am not sure I will do it properly. Can you do it?

**Comment by FareesHussain** [comment]
> Oh no. Prior PR merges caused two conflicts. They look easy to merge, however I am not sure I will do it properly. Can you do it?

@cpg Done, Please have a look.

**Comment by chirag-jn** [comment]
Please see the compilation issue @FareesHussain. In my opinion, it must have failed locally too.

File: src/main/res/values-te/strings.xml [hunk 3]
```diff
@@ -1,23 +1,51 @@
...
+    <string name="label_next">తదుపరి</string>
+    <string name="label_pause">మూసివేయి</string>
+    <string name="label_play">ఆన్ చేయండి</string>
+    <string name="label_previous">మునుపటి</string>
+    <string name="language_en">ఇంగ్లీష్</string>
+    <string name="language_fr">ఫ్రెంచ్</string>
+    <string name="language_es">స్పానిష్</string>
+    <string name="language_bn">బెంగాలీ</string>
+    <string name="language_de">జర్మన్</string>
    <string name="header_settings">సెట్టింగులు</string>
```
Comment (chirag-jn) [inline src/main/res/values-te/strings.xml]: `header_settings` is present twice.

**Review by chirag-jn — CHANGES_REQUESTED** [review]

File: src/main/res/values-te/strings.xml (hunk 3 above)
Comment (chirag-jn) [reply src/main/res/values-te/strings.xml]: Fixed the issue. Let's see CI.

Comment (FareesHussain) [reply src/main/res/values-te/strings.xml]: Thanks I've fixed the other conflicts too.

Comment (chirag-jn) [reply src/main/res/values-te/strings.xml]: The CI still fails. Try building the app locally.

Comment (FareesHussain) [reply src/main/res/values-te/strings.xml]: Yes, I've tried it There is some issue with punjabi language. I'm unable to figure it out.

Comment (FareesHussain) [reply src/main/res/values-te/strings.xml]: As my changes were initially minor shall I create a new PR for this

File: src/main/res/values/strings.xml [hunk 4]
```diff
@@ -173,6 +173,8 @@
 
     <string name="alert_delete_dialog">Delete file</string>
     <string name="alert_delete_confirm">Are you sure?</string>
+    <string name="never_opened">Never Opened</string>
+    <string name="last_opened">Last Opened</string>
```
Comment (chirag-jn) [inline src/main/res/values/strings.xml:178]: Please add `never_opened` and `last_opened` for `values-zh` (Chinese) too. Rest seems fine.

**Review by chirag-jn — CHANGES_REQUESTED** [review]

**Comment by cpg** [comment]
Did you guys get a chance to test this PR?
If so, how's it looking?

File: src/main/res/values/strings.xml (hunk 4 above)
Comment (FareesHussain) [reply src/main/res/values/strings.xml:178]: Sorry for the delay. I'll do it now

Comment (FareesHussain) [reply src/main/res/values/strings.xml:178]: Done please check

**Pull request closed without merging**